gems/sorbet-runtime/lib/types/private/class_utils.rb

```diff
@@ -94,7 +94,7 @@ def self.def_with_visibility(mod, name, visibility, method=nil, &block)
   # overriding it (if it is defined by one of mod's ancestors). Returns a ReplacedMethod instance
   # on which you can call `bind(...).call(...)` to call the original method, or `restore` to
   # restore the original method (by overwriting or removing the override).
-  def self.replace_method(mod, name, &blk)
+  def self.replace_method(mod, name, original_only=false, &blk)
```

Comment:
The comment above should be changed to `If original_only is false, returns a ReplacedMethod instance...`